syntax = "proto3";

import 'files/common/common.proto';

message SummonOddsLists {
  repeated SummonOddsList summon_odds_lists = 1;

  message SummonOddsList {
    ObjectIdentity id = 1;
    SummonOddsInfo info = 2;
  }

  message SummonOddsInfo {
    ObjectIdentity purchase_option = 1;
    ObjectIdentity item = 4;
    int32 quantity = 5;
    float probability = 6;
  }
}
